appserver/docker/src/main/java/fish/payara/docker/instance/admin/CreateDockerContainerCommand.java

```diff
@@ -173,6 +254,55 @@ public void execute(AdminCommandContext adminCommandContext) {
         }
     }
 
+    private WebTarget createWebTarget(Node node, String endpoint) {
+        Client client = ClientBuilder.newClient();
+        WebTarget webTarget = null;
+        if (Boolean.parseBoolean(node.getUseTls())) {
```

Comment:
Correct me if I'm wrong but could you do something like this to reduce redundency?;
```
boolean isHttps = Boolean.parseBoolean(node.getUseTls())
if(endpoint.startsWith("/")) {
    webTarget = client.target(isHttps ? "https://" : "http://" 
                  + node.getNodeHost()
                  + ":"
                  + node.getDockerPort()
                  + endpoint);
} else {
    webTarget = client.target(isHttps ? "https://" : "http://"
                  + node.getNodeHost()
                  + ":"
                  + node.getDocketPort()
                  + "/"
                  + endpoint);
}
```

Reply:
I'll give it a test - if I'm going with ternary form I might as well go whole hog with something like the following:
```
WebTarget webTarget = client.target((Boolean.parseBoolean(node.getUseTls()) ? "https://" : "http://")
                + node.getNodeHost()
                + ":"
                + node.getDockerPort()
                + (endpoint.startsWith("/") ? endpoint : "/" + endpoint));
```

Reply:
That works too, and still somewhat readable :P